core/src/main/kotlin/org/evomaster/core/problem/external/service/ExternalServiceRequest.kt

```diff
@@ -0,0 +1,44 @@
+package org.evomaster.core.problem.external.service
+
+import java.util.UUID
+
+/**
+ * Represent an external service call made to a WireMock
+ * instance from a SUT.
+ *
+ * TODO: Properties have to extended further based on the need
+ */
+class ExternalServiceRequest(
+    private val id: UUID,
+    private val method: String,
+    private val url: String,
+    private val absoluteURL: String,
+    private val wasMatched: Boolean,
+) {
+
```

Comment:
all the getters here seems redundant. just remove `private`. recall that `val` makes props final, ie, without setters